Test: 테스트 및 CI 적용

blingblin-g opened the pull request:
## 변경 사항

- 간단한 테스트를 붙였습니다.
  - `__test__` 디렉토리에 `***.test.js` 형식으로 만들면 됩니다.
- github action을 통해 테스트를 자동화하였습니다.
  - PR을 올릴 때마다 자동으로 테스트가 돌아갑니다.

## 변경한 이유
- 코드 퀄리티 향상을 위해

## 스크린샷 또는 관련 문서
```bash
yarn test
```

GulSam00 reviewed (approved):
확인했습니다.

euiminnn commented:
![image](https://user-images.githubusercontent.com/39411711/151650583-ff5be97c-8d06-4d0b-972c-d70fbd8b8510.png)

progress bar 배경부분이 회색(B4B4B4)이 더 좋을 것 같습니다.

첨부로 보내드리는게 0%일 때 상황입니다.

rkskekzzz reviewed (approved):
고생하셨습니다

blingblin-g committed:
Test: first test for loginButton component in TopNav

Co-authored-by: rockpell <aythffk@gmail.com>

rockpell committed:
Chore: setupTests 추가

- test 파일에서 매번 import 하지 않아도 되는 setupTests 파일 추가
- cra로 생성된 프로젝트는 별도의 설정 파일 없이 setupTest 파일만 있으면 적용 가능

blingblin-g committed:
Test: Rename LoginButton

blingblin-g committed:
Test: test ci 적용

PIut0 reviewed (approved):
CI 적용 및 테스트 코드를 돌리는 방법의 좋은 예시를 본거같습니다 ㅎㅎ

Skyrich2000 commented:
오홍 프론트는 이런식으로 테스트가 되는군요 짱신기하다

chichoon reviewed (approved):
테스팅 라이브러리 맨날 CRA 하면 디폴트로 설치되어있는거 한번도 안 써보고 넘겼는데 신기하네요
로그인버튼 예제 참고해서 시도해봐야겠군요,,,

GulSam00 commented:
수고하셨습니다. 한 번 해봐야겠네영
